articles/iot-edge/tutorial-nested-iot-edge.md

```diff
@@ -589,6 +628,21 @@ Notice that the image URI that we used for the simulated temperature sensor modu
 
 On the device details page for your lower layer IoT Edge device, you should now see the temperature sensor module listed along the system modules as **Specified in deployment**. It may take a few minutes for the device to receive its new deployment, request the container image, and start the module. Refresh the page until you see the temperature sensor module listed as **Reported by device**.
 
+## IotEdge check
+Iotedge check can be performed in nested hierarchy, even if the children doesn't have direct internet access.
+When running "iotedge check" from the lower layer the program will try to pull the image from the parent through port 443.
+In this How-to, we use port 8000, so we need to specify it:
+
+   ```bash
+   iotedge check --diagnostics-image-name <parent_device_fqdn_or_ip>:8000/azureiotedge-diagnostics:1.2.0-rc2
+   ```
+   
+The azureiotedge-diagnostics is pulled from the container registry that is linked with the registry module. This tutorial has it set by default to https://mcr.microsoft.com:
+    | Name | Value |
+    | - | - |
+    | `REGISTRY_PROXY_REMOTEURL` | `https://mcr.microsoft.com` |
+If you are using you own private container registry, make sure all the images (IoTEdgeAPIProxy, edgeAgent, edgeHub, diagnostics ...) are present in it.    
+    
```

Comment:
This section isn't very directive. You're telling people that they can run the check tool, but not why they should and what results they should be seeing. There's a possibility that users on this tutorial won't be familiar with the check command. 

I would rephrase this section to say something like, "Run the `iotedge check` command to verify xyz configuration/networking status or to troubleshoot any deployment errors." And then you can go on to explain how the check tool works within hierarchy scenarios.

Reply:
Added that line